### ISSUE TITLE
Accessibility Bug: Invalid aria-activedescendant="dropdownItem_-1" on Initial Open and missing aria attributes and roles

### ISSUE THREAD

**Issue by Faiz-exe** [primefaces/primereact#8004]
### Describe the bug

Initially, scanning the DropDown component with the axe DevTools accessibility scanner highlighted missing key accessibility attributes, including:
- role="combobox"
- aria-haspopup
- aria-controls
- aria-expanded

To address this, we manually added these attributes to the component. Notably, we had to introduce a useState hook to track the dropdown's visibility and correctly reflect it in aria-expanded:

```
const [isDropDownOpen, setDropDownOpen] = useState(false);

// Passed to the dropdown root as pt props:
pt : {
root: {
  role: 'combobox',
  'aria-haspopup': 'listbox',
  'aria-controls': 'statementLanguage-listbox',
  'aria-expanded': isDropDownOpen
}
}
```

However, during this process, a separate issue was identified: the component renders
**aria-activedescendant="dropdownItem_-1"** when the dropdown is initially opened — even though no option is focused. This ID does not exist in the DOM, which leads to a violation of ARIA specifications.

Here is a screenshot where axe devtools reports invalid aria attribute

![Image](https://github.com/user-attachments/assets/401e0155-821a-4675-834a-714c0f1bd4d3)

### System Information

```Shell
Primereact Version: 10.9.1
Browser: Chrome
Operating System: Windows
Browser: [e.g. Chrome, Firefox]
Accessibility tool used: Axe, axe-core, axe-cypress
```

### Steps to reproduce the behavior

1. Visit the official PrimeReact [DropDown documentation.](https://primereact.org/dropdown/)
2. Open your browser's developer tools and navigate to the Elements tab.
3. Locate the rendered  parnt <div> for the dropdown component.
4. Observe that the aria-activedescendant attribute is set to "dropdownItem_-1" even before any item is focused
5. Confirm that no DOM element with id="dropdownItem_-1" exists, making this an invalid ARIA reference.

![Image](https://github.com/user-attachments/assets/1bed86cb-b565-4fea-9ad2-dc9980839fad)

### Expected behavior

The aria-expanded attribute should ideally be managed internally by the component to accurately reflect its open/closed state. This would improve developer experience by eliminating the need to implement custom state management solely to meet accessibility requirements.

Avoid setting aria-activedescendant unless an item is actually focused, and ensure that when it is set, it references a valid DOM element rather than using a fallback like dropdownItem_-1.

**Comment by melloware** [comment]
PR is welcome.
